src/TextField/TextField.jsx

```diff
@@ -134,6 +134,12 @@ const TextField = React.createClass({
     style: React.PropTypes.object,
 
     /**
+     * Override the inline-styles of the TextField's textarea element.
+     * (only available when multiLine option is true)
```

Comment:
if multiLine is true the underlying element is a textarea (which also get inputStyle by the way), otherwize it's a simple input. (which do not use inputStyle).